app/views/movements/_form.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= form_with(model: movement, url: [@group, movement]) do |form| %>
+
+  <% if movement.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(movement.errors.count, "error") %> prohibited this movement from being saved:</h2>
+
+      <ul>
+        <% movement.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class='devise_login_register'>
+    <%= form.text_field :name, placeholder: 'Name' %>
+  </div>
+
+  <div class='devise_login_register'>
+    <%= form.text_field :amount, placeholder: 'Amount' %>
+  </div>
+
+  <div class=''>
+    <%= form.collection_check_boxes :group_ids, Group.where(user_id: current_user.id), :id, :name do |element| %>
+      <%= element.check_box + element.text %>
+    <% end %>
+  </div>
+  
+  <%# <div class='button_submit_transaction button_submit_form position-relative font_size_link_new container-fluid green_template text-white position-absolute bottom-0 start-50 translate-middle-x' style='height: 15%;'> %>
+
+    <%# <div class='button_submit_transaction button_submit_form font_size_link_new'> %>
+      <%# <%= form.submit 'Create transaction' %>
+    <%# </div> %>
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
@BenMukebo You are absolutely right, thank you very much for your comments. I have already applied your comments and applied your advice.

Reply:
This change has been done well